source/includes/_tipser-elements.md

```diff
@@ -102,6 +102,77 @@ All properties are optional:
 property name | description |type | default value
 --------------|-------------|-----|---------------
 defaultAddedToCartPopup | controls whether default Added To Cart Popup is displayed | boolean | true 
+eventsHandlers | gives possibility to add handlers to Tipser Elements events | object | empty
+useDefaultErrorHandler | if set to false there will be no error message displayed in case of error | boolean | true
+
+#### Event Handlers
+Event handlers may be passed as a part of `config` option of `TipserElementsProvider`. There is a number of events exposed for developer. Each of the handler may have a different params as arguments of handler. 
+
+```js
+
+let tipserConfig = {
+    lang: 'en',
+    primaryColor: 'red',
+    // ----- EVENT HANDLING START
+    useDefaultErrorHandler: true,
+    eventsHandlers: {
+        onError: (error) => {
+            console.log(error)
+        },
+        onAddToCart: ({cartSize, product}) => {
+            console.log('Hurray, you have added item to cart. ', product);
+            console.log('Your cart size is now. ', cartSize);
+        }
+    }
+    // -- EVENT HANDLING END
+};
+
+class App extends Component {
+
+    render() {
+        return (
+            <Router history={hashHistory}>
+                <TipserElementsProvider
+                    posId={"5075d7715c3d090a90585e87"}
+                    config={tipserConfig}>
+                    <Switch>
+                        <Route path="/product/:productId" component={ProductView} />
+                        <Route path="/" component={ComponentsView} />
+                        <Route component={NotFoundView} />
+                    </Switch>
+                </TipserElementsProvider>
+            </Router>
+        );
+    }
+
+}
+
+```
+
+Handler name  | description | params
+--------------|-------------|--------
+onError | Main goal of this handler is to add additional behavior when the error appears. | object of type EventError ```js export interface EventError {
```

Comment:
this does not look good when md is compiled:

<img width="1029" alt="Zrzut ekranu 2019-05-9 o 11 45 28" src="https://user-images.githubusercontent.com/3996881/57444881-71486c80-7251-11e9-9f52-ea7fd8e4e3c8.png">

I would suggest to write interface somewhere below, and put a link or reference to it in `params` column

Reply:
I agree, something is wrong here :(

Reply:
see now :)